lib/supa/commands/attributes.rb

```diff
@@ -0,0 +1,18 @@
+module Supa
+  module Commands
+    class Attributes < Supa::Command
```

Comment:
@Strech It is possible but it was @dasnotme's request to implement it first without any options.